### PR TITLE
tp-qemu: update balloon test boundary value

### PR DESCRIPTION
**Pull request by xutian**
Signed-off-by: Xu Tian <xutian@redhat.com>

### CODE REVIEW

**Comment by xutian** [comment]
deps on https://github.com/avocado-framework/avocado-vt/pull/366

**Comment by xutian** [comment]
deps on https://github.com/avocado-framework/avocado-vt/pull/359

**Comment by xutian** [comment]
ID:  1146816

**Comment by maxujun** [comment]
Acked-by: Xujun Ma xuma@redhat.com

File: qemu/tests/balloon_check.py [hunk 1]
```diff
@@ -86,14 +78,12 @@ def balloon_memory(self, new_mem):
         """
         error.context("Change VM memory to %s" % new_mem, logging.info)
         compare_mem = new_mem
-        if self.params["monitor_type"] == "qmp":
-            new_mem = new_mem * 1024 * 1024
-        # This should be replaced by proper monitor method call
-        self.vm.monitor.send_args_cmd("balloon value=%s" % new_mem)
+        self.vm.balloon(new_mem)
```
Comment (CongSmile) [inline qemu/tests/balloon_check.py]: vm.monitor.balloon()

**Comment by CongSmile** [comment]
Update to vm.monitor.balloon(), and test failed.

20:08:29 DEBUG| Sending command: wmic OS get FreePhysicalMemory
20:08:29 ERROR| Memory size mismatch after evict memory:

20:08:29 ERROR| Wanted to be changed: 1282
Changed in monitor: 1286
Changed in guest: 1468

File: qemu/tests/balloon_check.py (hunk 1 above)
Comment (xutian) [inline qemu/tests/balloon_check.py]: qemu_vm.py has balloon interface named balloon(), so above line it right.

**Comment by xutian** [comment]
Very strange, there has 200M difference, which guest you are tested? can you show me log file.

Thanks,
Xu

**Comment by xutian** [comment]
@CongSmile  336 has updated, can you help retest this PR.

Thanks,
Xu

**Comment by MiriamDeng** [comment]
@xutian 
   Re-test this patch but hit the following error log - 
 13:25:03 DEBUG| Searching for test modules that match 'type = balloon_check' and 'provider = io-github-autotest-qemu' on this cartesian dict
13:25:03 DEBUG| Found subtest module /usr/share/avocado/data/avocado-vt/test-providers.d/downloads/io-github-autotest-qemu/qemu/tests/balloon_check.py
13:25:03 ERROR|
13:25:03 ERROR| Reproduced traceback from: /usr/lib/python2.7/site-packages/avocado_plugins_vt-0.34.0-py2.7.egg/avocado_vt/test.py:400
13:25:03 ERROR| Traceback (most recent call last):
13:25:03 ERROR|   File "/usr/lib/python2.7/site-packages/avocado_plugins_vt-0.34.0-py2.7.egg/avocado_vt/test.py", line 176, in runTest
13:25:03 ERROR|     self._runTest()
13:25:03 ERROR|   File "/usr/lib/python2.7/site-packages/avocado_plugins_vt-0.34.0-py2.7.egg/avocado_vt/test.py", line 285, in _runTest
13:25:03 ERROR|     test_modules[t_type] = imp.load_module(t_type, f, p, d)
13:25:03 ERROR|   File "/usr/share/avocado/data/avocado-vt/test-providers.d/downloads/io-github-autotest-qemu/qemu/tests/balloon_check.py", line 8, in <module>
13:25:03 ERROR|     from virttest.utils_test.qemu import MemoryBaseTest
13:25:03 ERROR| ImportError: cannot import name MemoryBaseTest
13:25:03 ERROR|
13:25:03 ERROR| Traceback (most recent call last):

13:25:03 ERROR|   File "/usr/lib/python2.7/site-packages/avocado_plugins_vt-0.34.0-py2.7.egg/avocado_vt/test.py", line 414, in _run_avocado
    raise test_exception

13:25:03 ERROR| ImportError: cannot import name MemoryBaseTest

13:25:03 ERROR| ERROR smp_2.4096m.repeat1.run_test.Host_RHEL.m7.u3.qcow2.virtio_scsi.up.virtio_net.RHEL.7.1.x86_64.io-github-autotest-qemu.balloon_check.balloon_evict.balloon-migrate -> ImportError: cannot import name MemoryBaseTest

**Comment by xutian** [comment]
@MiriamDeng  
It looks your avocado-vt source code not update, please pull your code try again.

Thanks,
Xu

**Comment by MiriamDeng** [comment]
Acked-by: mdeng@redhat.com.
@xutian thanks a lot.

**Comment by xutian** [comment]
thanks @MiriamDeng
